Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -185,11 +184,11 @@ void SVGFormattingContext::run(AvailableSpace const& available_space)
                                            .top_level_browsing_context() // ...in the browser
                                            .active_document()            // the active document (in the browser)
                                            ->is_initial_about_blank();   // is it about:blank initially?
-    //  NOTE: Given that somehow we have already made it to an SVGFormattingContext:
```

Comment:
` LibWeb: Correct formatting issues`

should be fixup'd into the first commit.